### PR TITLE
docs: replace certificates in pkcs12 format

### PR DESCRIPTION
**Pull request by PaulRMellor**
Signed-off-by: prmellor <pmellor@redhat.com>

**Documentation**
Updates the _Installing your own CA certificates_ procedure in the _Using Strimzi_ guide.
The procedure now includes recommendation and updated steps to create cluster certificates in PKCS12 format when replacing the secret generated by the Cluster Operator. Applies specifically when connecting to service registry. 

- [ ] Write tests
- [ ] Make sure all tests pass
- [x] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [ ] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md
- [ ] Supply screenshots for visual changes, such as Grafana dashboards

### CODE REVIEW

File: documentation/modules/security/proc-installing-your-own-ca-certificates.adoc [hunk 1]
```diff
@@ -22,49 +32,93 @@ The chain should be in the following order:
...
+
+.Before you begin
+The Cluster Operator generates the following files for the `_CLUSTER-NAME_-cluster-ca-cert` secret:
+
+* `ca.crt` cluster certificate in PEM format
+* `ca.p12` cluster certificate in PKCS #12 format
+* `ca.password` to access the PKCS #12 file
+
+If you are using a service registry, make sure you include all three files in your secret.
+The service registry can then use a certificate in the appropriate format to connect securely with Kafka.
```
Comment (scholzj) [inline documentation/modules/security/proc-installing-your-own-ca-certificates.adoc:46]: I don't think this should be here. Not all service registries require PKCS12 files and not all applications which need PKCS12 files are service registries. Maybe we can use something like this?

```suggestion
Some applications cannot use the PEM certificates directly and support only the PKCS #12 format.
To make it easier to use them, you can also add the cluster certificate in PKCS #12 format.
```

Comment (PaulRMellor) [reply documentation/modules/security/proc-installing-your-own-ca-certificates.adoc:46]: Better. I thought I needed to say something specifically about service registries. 
Changed to:

> Some applications cannot use PEM certificates and support only PKCS #12 certificates. You can also add your own cluster certificate in PKCS #12 format.

**Review by scholzj — APPROVED** [review]

File: documentation/modules/security/proc-installing-your-own-ca-certificates.adoc [hunk 2]
```diff
@@ -5,9 +5,19 @@
...
 = Installing your own CA certificates
 
+[role="_abstract"]
 This procedure describes how to install your own CA certificates and keys instead of using the CA certificates and private keys generated by the Cluster Operator.
 
-You can use this procedure to install your own cluster or client CA certificates.
+The Cluster Operator automatically generates and renews the following secrets:
+
+`_CLUSTER-NAME_-cluster-ca`:: The cluster secret that contains the private key for the cluster CA.
+`_CLUSTER-NAME_-cluster-ca-cert`:: The cluster secret contains a cluster CA certificate. The certificate contains a public key to validate the identity of Kafka brokers.
```
Comment (laidan6000) [inline documentation/modules/security/proc-installing-your-own-ca-certificates.adoc:14]:
```suggestion
`_CLUSTER-NAME_-cluster-ca-cert`:: The cluster secret that contains a cluster CA certificate. The certificate contains a public key to validate the identity of Kafka brokers.
```

File: documentation/modules/security/proc-installing-your-own-ca-certificates.adoc [hunk 3]
```diff
@@ -5,9 +5,19 @@
...
+`_CLUSTER-NAME_-clients-ca`:: The client secret that contains the private key for the client CA.
+`_CLUSTER-NAME_-clients-ca-cert`:: The client secret contains a client CA certificate. The certificate contains a public key to validate the identity of clients accessing the Kafka brokers.
```
Comment (laidan6000) [inline documentation/modules/security/proc-installing-your-own-ca-certificates.adoc:16]:
```suggestion
`_CLUSTER-NAME_-clients-ca-cert`:: The client secret that contains a client CA certificate. The certificate contains a public key to validate the identity of clients accessing the Kafka brokers.
```

**Review by laidan6000 — APPROVED** [review]
LGTM

**Pull request merged**